src/app/public/modules/lookup/lookup-show-more-modal.component.html

```diff
@@ -0,0 +1,84 @@
+<sky-modal>
+  <sky-modal-header>
+    {{getModalTitle()}}
```

Comment:
Using methods inside component templates [can cause performance problems](https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496). Possible to use a static `modalTitle` property instead?

Reply:
I actually just moved this logic to the template. I'm glad you caught this though because when I looked I realized I forgot to add the i18n strings for this title. I think in this case that the template makes more sense for this logic as having it in the component would make us deal with the i18n service which is much more gross than the pipe in the template.

Reply:
Your solution works well.